dbschema/1.tables.sql

```diff
@@ -20,3 +20,22 @@ CREATE INDEX tbl_token_transfers_value ON tbl_token_transfers (value);
 CREATE INDEX tbl_token_transfers_block_date ON tbl_token_transfers (block_date);
 CREATE INDEX tbl_token_transfers_block_number ON tbl_token_transfers (block_number);
 CREATE INDEX tbl_token_transfers_block_hash ON tbl_token_transfers (block_hash);
+
+CREATE TABLE tbl_token_metadata (
+    id BIGSERIAL PRIMARY KEY,
+    created_date TIMESTAMP WITHOUT TIME ZONE NOT NULL
+    updated_date TIMESTAMP WITHOUT TIME ZONE NOT NULL
+    registry_address TEXT NOT NULL,
+    token_id TEXT NOT NULL,
+    metadata_url TEXT NOT NULL,
+    image_url TEXT NOT NULL
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    attributes TEXT NOT NULL
+
+
+);
+CREATE UNIQUE INDEX tbl_token_registry_address_token_id ON tbl_token_metadata (registry_address, token_id);
```

Comment:
the names of these indices (the part after CREATE UNIQUE INDEX) should follow a formault of `<table_name>_<field_name_1>_<field_name_2>_...`

Reply:
this name is still wrong